docker/Dockerfile.test

```diff
@@ -2,7 +2,7 @@
 FROM rust:latest as builder
 
 RUN rustup default nightly-2020-02-18 && rustup component add rustfmt
-RUN apt-get update && apt-get install -y unzip
+RUN apt-get update && apt-get install -y unzip libssl-dev
```

Comment:
What's pulling libssl in?

Reply:
I was trying to figure out why running tests with TLS did not work. I thought the Docker crash was caused by a missing dependency. It turned out it was a crash caused by the RAM limit of my Docker configuration. I removed it.